flake.nix

```diff
@@ -1,38 +1,34 @@
 {
   description = "Secret management with age";
-
-  inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-  };
```

Comment:
IMVHO, we should keep this explicit input. It's a pet peeve of mine that people rely on the implicit `nixpkgs` input, because it's harder to override / change / update, as well as can be confusing if you don't realize that it's there right away.

Reply:
Alright, put this back in.